### PR TITLE
fix: account unification modals

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* fix: account unification modals
* fix: updated words
* fix: display the 'unify accounts' button only on Shibuya
* fix: updated connecting metamask logic
* feat: added a check input for sending submit transaction

ref: https://github.com/AstarNetwork/astar-apps/pull/925#issuecomment-1744877278  

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes
- [ ] tested on mobile devices

**This pull request makes the following changes:**

**Adds**
* feat: added a check input for sending submit transaction
image

### CODE REVIEW

**Review by bobo-k2 — APPROVED** [review]

File: src/i18n/en-US/index.ts [hunk 1]
```diff
@@ -190,6 +190,7 @@ export default {
         'If you already have an account ID that is created on Polkadot.js app, this account unification information will override.',
       agreeToProceed:
         'I have read all and I would like to proceed to create a unified account please.',
+      agreeToSubmit: 'I understand this action will not be canceled.',
```
Comment (bobo-k2) [inline src/i18n/en-US/index.ts:193]: Maybe to provide a bit more explanation, something like to add: Once accounts are unified they can't be separated. 
Probably need a better word than separate :)

Comment (impelcrypto) [reply src/i18n/en-US/index.ts:193]: Maybe `I understand that once accounts have been unified, they cannot be separated.` ?  @gluneau @Kahonnohak @ayumitk Please advise 🙏

Comment (impelcrypto) [reply src/i18n/en-US/index.ts:193]: image

Comment (Kahonnohak) [reply src/i18n/en-US/index.ts]: > Maybe `I understand that once accounts have been unified, they cannot be separated.` ? @gluneau @Kahonnohak @ayumitk Please advise 🙏

happy with it!

**Review by Kahonnohak — APPROVED** [review]
LGTM

**Pull request merged**